### DISCUSSION TITLE
Broken syntax highlighting when variable is at the beginning of line.

### DISCUSSION THREAD

**Discussion by theinfinit** [mdx-js/mdx#2426] [Q&A] [answered]
Context:
- Astro Framework
- Presence of frontmatter doesn't affect the issue
- Issue also occurs with default VS Code color theme

When I use variable at the beginning of the line, syntax highlighting breaks. Screenshots below:

Broken:
![Screenshot 2024-01-10 133820](https://github.com/mdx-js/mdx/assets/33784221/daa16b69-836e-4728-87ea-7fdd6db39d0e)

Correct:
![Screenshot 2024-01-10 133904-works](https://github.com/mdx-js/mdx/assets/33784221/9eaea4c4-f932-4ce9-8203-b4525b12c1a2)

Is that bug or my misconfiguration of sorts? I searched for similar issues/discussions, but couldn't find anything. 

I will be glad for your assistance 🙏

**Comment by remcohaszing** [comment, marked as answer]
This is a bug in https://github.com/wooorm/markdown-tm-language. I noticed this too, but I see this hasn't been reported yet.
